modal/shared/test_config.py

```diff
@@ -0,0 +1,29 @@
+import os
+from typing import Annotated
+
+from fastapi import Depends, FastAPI, testclient
+from fastapi.security import HTTPAuthorizationCredentials
+
+from shared.config import Config
+
+
+def test_auth():
+    """The API auth dependency should prevent unauthorized requests."""
```

Comment:
test works locally, but i didn't wanna overload the PR with a new CI runner just yet --  happy to add it in now or as a followup if ya want?

```
$ pytest modal/shared

========= test session starts =========
platform linux -- Python 3.10.12, pytest-7.4.3, pluggy-1.3.0
rootdir: /home/sam/dev/openrouter-runner
plugins: anyio-3.7.1
collected 1 item                                                                                             

modal/shared/test_config.py .                                                                          [100%]

========= 1 passed in 0.56s =========
```

Reply:
ehh it was easy enough so i went ahead with it, feel free to close or merge it into here if ya want

Reply:
@sambarnes I think the CI PR targeted the wrong remote (or wrong github org)

Reply:
Yah I had it based off of this auth-refactor branch. So I guess GitHub treats it differently if it's two branches on my fork in the PR 🤷‍♀️ lol

I just merged it into here instead